_includes/scripts/misc.html

```diff
@@ -10,6 +10,32 @@
   <script defer src="{{ '/assets/js/zoom.js' | relative_url }}"></script>
 {%- endif -%}
 
+  <!-- Bootstrap Table -->
+  <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.21.3/dist/bootstrap-table.min.css">
+  <script defer src="https://unpkg.com/bootstrap-table@1.21.3/dist/bootstrap-table.min.js"></script>
+  <script>
+    $(document).ready(function() {
+      $('table').each(function() {
+        // only select tables that are not inside an element with "news" (about page) or "card" (cv page) class
+        if($(this).parents('[class*="news"]').length==0 &&
+            $(this).parents('[class*="card"]').length==0 &&
+            $(this).parents('code').length == 0) {
+          // make table use bootstrap-table
+          $(this).attr('data-toggle','table');
+          // add some classes to make the table look better
+          // $(this).addClass('table-sm');
+          $(this).addClass('table-hover');
+
+          if (document.documentElement.getAttribute("data-theme") == "dark") {
+            $(this).addClass('table-dark');
+          } else {
+            $(this).removeClass('table-dark');
+          }
+        }
+      })
+    });
```

Comment:
this file is supposed to just include various scripts. let's move this code to `assets/js/common.js`.

Reply:
When I moved this to `assets/js/common.js` it stopped stylizing the markdown table, dunno why.

![image](https://user-images.githubusercontent.com/31376482/230723894-c85b61df-4584-4803-b504-eeb982aff2f5.png)

Reply:
Apparently after updating to ruby 3.2.2 this doesn't happen anymore. Updating code and merging it.

Reply:
Don't know why this happens yet, but since this is not actually an impediment, what do you think of using it this way @alshedivat?

Reply:
fixed